crates/worker/src/dynamodb_client.rs

```diff
@@ -0,0 +1,47 @@
+use crate::errors::DBError;
+use aws_sdk_dynamodb::types::AttributeValue;
+use aws_sdk_dynamodb::Client;
+use types::item::Item;
+
+#[derive(Clone)]
+pub struct DynamoDBClient {
+    pub client: Client,
+    pub table_name: String,
```

Comment:
Well, we could use  Box::leak() to make table_name a `&'static str`, but that wouldn't really help, since the issue is in table_name interface which clones the value anyway.
`pub fn table_name(mut self, input: impl ::std::convert::Into<::std::string::String>) -> Self`

Having `pub table_name: &str` would bound DynamoDBClient lifetime to table_name that isn't really correct